webapp/src/components/meeting_settings/meeting_settings.jsx

```diff
@@ -17,8 +17,9 @@ export default class MeetingSettingsModal extends React.PureComponent {
         super(props);
 
         this.state = {
-            hashtag: '{{Jan02}}',
+            hashtagPrefix: 'Prefix',
```

Comment:
Do we have a way for the user to define their own date format that is used, like the feature currently works?

Reply:
no, as per issue. IMO it is end goal to limit that.